packer/configure-windows-vhd.ps1

```diff
@@ -37,18 +37,54 @@ function Disable-WindowsUpdates
     Set-ItemProperty -Path $AutoUpdatePath -Name NoAutoUpdate -Value 1 | Out-Null
 }
 
-
 function Get-ContainerImages
 {
     $imagesToPull = @(
         "mcr.microsoft.com/windows/servercore:ltsc2019",
         "mcr.microsoft.com/windows/nanoserver:1809",
         "mcr.microsoft.com/k8s/core/pause:1.2.0")
 
+    foreach ($image in $imagesToPull) {
+        docker pull $image
+    }
+}
+
+function Get-FilesToCacheOnVHD
+{
+    Write-Log "Caching misc files on VHD"
+
+    $map = @{
+        "c:\akse-cache\" = @(
+            "https://github.com/Microsoft/SDN/raw/master/Kubernetes/flannel/l2bridge/cni/win-bridge.exe",
+            "https://github.com/Microsoft/SDN/raw/master/Kubernetes/windows/hns.psm1"
+        );
+        "c:\akse-cache\win-k8s\" = @(
+            "https://acs-mirror.azureedge.net/wink8s/azs-v1.14.5-1int.zip",
+            "https://acs-mirror.azureedge.net/wink8s/azs-v1.14.6-1int.zip",
+            "https://acs-mirror.azureedge.net/wink8s/azs-v1.15.2-1int.zip",
+            "https://acs-mirror.azureedge.net/wink8s/azs-v1.15.3-1int.zip"
+            "https://acs-mirror.azureedge.net/wink8s/v1.14.5-1int.zip",
+            "https://acs-mirror.azureedge.net/wink8s/v1.14.6-1int.zip",
+            "https://acs-mirror.azureedge.net/wink8s/v1.15.2-1int.zip",
+            "https://acs-mirror.azureedge.net/wink8s/v1.15.3-1int.zip"
+        );
+        "c:\akse-cache\win-vnet-cni\" = @(
+            "https://acs-mirror.azureedge.net/cni/azure-vnet-cni-windows-amd64-v1.0.27.zip"
+        )
+    }
 
-    foreach ($image in $imagesToPull)
+    foreach ($dir in $map.Keys)
     {
-        docker pull $image
+        New-Item -ItemType Directory $dir -Force | Out-Null
+
+        foreach ($URL in $map[$dir])
+        {
+            $fileName = [IO.Path]::GetFileName($URL)
+            $dest = [IO.Path]::Combine($dir, $fileName)
+
+            Write-Log "Downloading $URL to $dest"
+            Invoke-WebRequest -UseBasicParsing -Uri $URL -OutFile $dest

```

Comment:
isn't code similar to below needed for downloading the artifacts.

$secureProtocols = @()
        $insecureProtocols = @([System.Net.SecurityProtocolType]::SystemDefault, [System.Net.SecurityProtocolType]::Ssl3)
        $oldProgressPreference = $ProgressPreference
        $ProgressPreference = 'SilentlyContinue'
        Invoke-WebRequest $Url -UseBasicParsing -OutFile $DestinationPath -Verbose
        $ProgressPreference = $oldProgressPreference
        Write-Log "Downloaded file to $DestinationPath"

Reply:
I'll check the default behavior and update appropriately :)